### ISSUE TITLE
Featured in block

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/student-loan-planner-theme#10]
Missing alternative text. should pull from the image alt text field.
![Screenshot 2023-09-05 at 2 26 37 PM](https://github.com/equalizedigital/student-loan-planner-theme/assets/2895788/7b353343-254f-48c7-ad8b-61a543feeb46)

Basecamp Ref: https://3.basecamp.com/3579237/buckets/32176113/card_tables/cards/6504311905
